<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="SoftwareManagement.sm_apt_install_download_path" skipped="0" tests="1" time="85.63" timestamp="2022-06-20T17:12:09">
	<testcase classname="AptInstallWithDownloadPath" name="SoftwareManagement.sm_apt_install_download_path">
		<failure message="BLOCKED: AssertionError:" type="BLOCKED"/>
		<system-out>17:10:44 INFO  ==============================================================
17:10:44 INFO  Id:    SoftwareManagement.sm_apt_install_download_path
17:10:44 INFO  Dir:   software_management_end_to_end/
17:10:44 INFO  Title: Install package to directory specified in tedge config set tmp.path
17:10:44 INFO  ==============================================================
17:10:44 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
17:10:44 INFO  EnvironmentC8y Setup
17:10:45 INFO  Executed systemctl&lt;serv_mapper1&gt;, exit status 3
17:10:59 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
17:11:01 INFO  Executed sudo&lt;tedge_connect_c8y_test&gt;, exit status 0
17:11:02 INFO  Executed systemctl&lt;serv_mosq2&gt;, exit status 0
17:11:03 INFO  Executed systemctl&lt;serv_mapper3&gt;, exit status 0
17:11:05 ERROR No operations found, assuming it passed
17:11:05 INFO  Assert that (False == value) with value=False ... passed
17:11:06 INFO  Executed sudo, exit status 0
17:11:06 INFO  Executed sudo, exit status 0
17:11:07 INFO  Executed sudo, exit status 0
17:11:11 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
17:11:11 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
17:11:11 INFO  Delaying execution by 7 seconds
17:11:32 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 13 secs
17:11:32 INFO  Response status: 201
17:11:32 INFO  Response to action: {
    &quot;creationTime&quot;: &quot;2022-06-20T17:11:32.381Z&quot;,
    &quot;deviceName&quot;: &quot;m32sd10c&quot;,
    &quot;deviceId&quot;: &quot;20352405&quot;,
    &quot;self&quot;: &quot;https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20362633&quot;,
    &quot;id&quot;: &quot;20362633&quot;,
    &quot;status&quot;: &quot;PENDING&quot;,
    &quot;description&quot;: &quot;Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]&quot;,
    &quot;c8y_SoftwareUpdate&quot;: [
        {
            &quot;name&quot;: &quot;rolldice&quot;,
            &quot;action&quot;: &quot;install&quot;,
            &quot;id&quot;: &quot;5152439&quot;,
            &quot;version&quot;: &quot;::apt&quot;,
            &quot;url&quot;: &quot;https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463&quot;
        }
    ]
}
17:11:32 INFO  Started operation: {'creationTime': '2022-06-20T17:11:32.381Z', 'deviceName': 'm32sd10c', 'deviceId': '20352405', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/20362633', 'id': '20362633', 'status': 'PENDING', 'description': &quot;Apply software changes, triggered from PySys: [{'name': 'rolldice', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463', 'action': 'install'}]&quot;, 'c8y_SoftwareUpdate': [{'name': 'rolldice', 'action': 'install', 'id': '5152439', 'version': '::apt', 'url': 'https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/12435463'}]}
17:11:32 WARN  , retrying in 0.5 seconds...
17:11:32 WARN  , retrying in 0.5 seconds...
17:11:33 WARN  , retrying in 0.5 seconds...
17:11:33 WARN  , retrying in 0.5 seconds...
17:11:34 WARN  , retrying in 0.5 seconds...
17:11:34 WARN  , retrying in 0.5 seconds...
17:11:35 WARN  , retrying in 0.5 seconds...
17:11:35 WARN  , retrying in 0.5 seconds...
17:11:36 WARN  , retrying in 0.5 seconds...
17:11:36 WARN  AssertionError occurred while running test - 
Traceback (most recent call last):
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py&quot;, line 1294, in __call__
    self.testObj.execute()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 83, in execute
    assert_install_in_download_path(install_directory=self.DOWNLOAD_DIR)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/decorator.py&quot;, line 232, in fun
    return caller(func, *(extras + args), **kw)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py&quot;, line 74, in retry_decorator
    logger)
  File &quot;/home/pi/env-pysys/lib/python3.7/site-packages/retry/api.py&quot;, line 33, in __retry_internal
    return f()
  File &quot;/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_apt_install_download_path/run.py&quot;, line 28, in assert_install_in_download_path
    assert &quot;rolldice&quot; in os.listdir(f&quot;{install_directory}&quot;)
AssertionError
17:11:37 WARN  AssertionError: ... blocked 
17:11:37 INFO  
17:11:37 INFO  cleanup:
17:11:38 INFO  Executed sudo, exit status 0
17:11:38 INFO  Executed sudo, exit status 0
17:11:42 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
17:11:43 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 3 seconds
17:11:43 INFO  Delaying execution by 8 seconds
17:12:05 INFO  Executed sudo&lt;tedge_connect_c8y&gt;, exit status 0, duration 14 secs
17:12:09 INFO  Executed sudo&lt;tedge_disconnect_c8y&gt;, exit status 0
17:12:09 INFO  Executed systemctl&lt;serv_mapper5&gt;, exit status 3
17:12:09 INFO  
17:12:09 INFO  Test duration: 85.63 secs
17:12:09 INFO  Test final outcome:  BLOCKED
17:12:09 INFO  Test outcome reason: AssertionError:
17:12:09 INFO  
</system-out>
	</testcase>
</testsuite>
